frontend/src/pages/Dashboard/Dashboard.tsx

```diff
@@ -231,11 +233,10 @@ const NextSessionButton = styled(PurpleButton)`
 
 const WeekProgressContainer = styled.div`
   flex: 2;
-  min-height: 100%;
 
```

Comment:
Can we use flex here so that we can do `justify-content: flex-end` ?  "this week + container" and the "your progress + container" are the children elements (they are 4 separate elements as of right now, but can wrap them in divs to group two)

I think it would look better if the "this week + container" was aligned with the bottom of the screen

Reply:
Sorry disregard this. I don't know if this comment accurately represents Stephanie's design